### PR TITLE
Delay installed telemetries

### PR DESCRIPTION
**Pull request by ajhsu**
## Problems to solve
The reported installed numbers from GA has a huge gap from numbers from Chrome Web Store.

## Observations and trials
After a series of trials, it seems that the `event_callback` cannot ensure that GTM requests are being sent; Instead, I tried to delay the redirection until the entire page is loaded, and it seems to work.

## Changes
1. Delay the redirection until the entire page is loaded.
2. Set the debounce timer to avoid calling "event_callback" multiple times. The trade-off here is that it increases 1s delay for this redirection page.
3. Remove unused resources to improve the loading speed.
4. Set an idle timer (10s) to perform the redirection. This should prevent from stuck at the redirecting page due to adblockers.

### CODE REVIEW

File: src/installed.html [hunk 1]
```diff
@@ -10,18 +10,23 @@
...
-    $(function() {
-      var whatsAppWebLink = 'https://web.whatsapp.com/'
+    <script>
+      window.addEventListener('load', function onPageLoaded() {
+        var whatsAppWebLink = 'https://web.whatsapp.com/';
+        var debounceTimer = null;
+        var timeout = 1000;
+        gaTracking(
+          'installed_extension', 'click', 'installPage', null,
+          function (id) {
```
Comment (ajhsu) [inline src/installed.html:20]: It's worth to mention that the callback function will be fired 4 times due to we have multiple GTM containers. So I tried to aggregate the requests into one by setting a debounce timer here.

**Review by leokuo0724 — COMMENTED** [review]
Should we also delay ga tracking in `install.html`, `subscribed.html` and `uninstall-extension.html` pages?

**Review by leokuo0724 — APPROVED** [review]
I tested it and works well! Thanks @ajhsu

**Comment by ajhsu** [comment]
> Should we also delay ga tracking in `install.html`, `subscribed.html` and `uninstall-extension.html` pages?

Yes. We should apply to all redirecting pages. Will send another PR for them. Thanks for reminding!

**Pull request merged**